pkgs/applications/virtualization/virtualbox/gcc-13.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp b/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp
+index 8a9d15c0..260ebc5c 100644
+--- a/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp
++++ b/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp
+@@ -46,6 +46,9 @@
+ #if defined(VBOX_VBGLR3_XFREE86)
+ extern "C" {
+ # define XFree86LOADER
++# ifdef RT_GNUC_PREREQ(13,0) /* cmath gets dragged in and the c++/13/cmath header is allergic to -ffreestanding.  */
++#  define _GLIBCXX_INCLUDE_NEXT_C_HEADERS
++# endif
```

Comment:
Is there some reference for this?  It seems rather magical.

Reply:
Yeah, I just copied it from upstream: https://www.virtualbox.org/browser/vbox/trunk/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp?rev=99876#L49